spec/rails_helper.rb

```diff
@@ -1,5 +1,5 @@
-require 'simplecov'
-SimpleCov.start
+require 'coveralls'
+Coveralls.wear!
```

Comment:
I checked coveralls' docs and it looks like they say they recommend `Coveralls.wear!('rails')` for rails projects. My apologies if I'm misunderstanding something.